tests/www/apply/test_list_for_siae.py

```diff
@@ -32,20 +32,18 @@
 from tests.users.factories import JobSeekerFactory
 from tests.utils.htmx.test import assertSoupEqual, update_page_with_htmx
 from tests.utils.test import (
-    TestCase,
     assert_previous_step,
     assertSnapshotQueries,
     get_rows_from_streaming_response,
     parse_response_to_soup,
 )
 
 
-@pytest.mark.usefixtures("unittest_compatibility")
-class ProcessListSiaeTest(TestCase):
+class TestProcessListSiae:
     SELECTED_JOBS = "selected_jobs"
 
-    @classmethod
-    def setUpTestData(cls):
```

Comment:
Ce changement coûte 19 (tests) * ~0.5s, soit une dizaine de secondes. À voir si garder unitest pour ce cas ne se justifierait pas.

Reply:
En local j'ai plutôt 4 secondes de différence sur la class de tests (c'est bien le seul endroit où j'ai vu une différence significative)
Étant donné que je n'abandonne pas espoir qu'on arrive à migrer les notifications, ce serait le seul endroit avec un TestCase donc la question se pose.
Je propose de demander au public

Reply:
Effectivement, je ne mesure que 3s de différence (10s vs 7s sur un best of 3).

Reply:
Je vais merger sans ces tests là, je ferrais si j'arrive à accélérer un peu la suite de tests